action.yaml

```diff
@@ -29,6 +29,10 @@ inputs:
   docker-build-path:
     description: The Dockerfile path to use for the build
     required: false
+  docker-platform:
+    description: The target architecture for the image build
+    required: false
+    default: "linux/amd64"
```

Comment:
Consider having a default of `""`. This will allow docker to build with the correct architecture for the runner which would be a preferred default behavior.

Reply:
Happy for it to be `default: "linux/amd64"`. Accompanying documentation mentions the preferred default arch.